test/fuzz/CMakeLists.txt

```diff
@@ -45,6 +49,36 @@ target_link_libraries(
 
 set(FUZZ_TEST_TARGETS "")
 
+set(DEFAULT_RUNS_NUMBER 10)
+
+string(JOIN " " DEFAULT_LIBFUZZER_OPTS
+    -max_total_time=60
+    -mutate_depth=20
+    -print_final_stats=1
+    -print_pcs=1
+    -reduce_inputs=1
+    -reload=1
+    -report_slow_units=5
+    -use_value_profile=1
+    -workers=${CMAKE_BUILD_PARALLEL_LEVEL}
+    -runs=$\\{RUNS:-${DEFAULT_RUNS_NUMBER}\\}
```

Comment:
I don't get this line.
What amount of runs do you mean?
-1 means infinite amount of runs, but we trying to set -10?
Also, why don't set it like the following:
```
-runs=-${DEFAULT_RUNS_NUMBER}
```

Reply:
This is a parameter expansion in Bash, see [^1]. By default, the number of runs for a fuzzing test is infinite. I set explicitly it to 10, if env variable `RUNS` is empty and to `$RUNS` if it is not.

[^1]. https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html